posthog-react-native/test/posthog.spec.ts

```diff
@@ -523,4 +523,90 @@ describe('PostHog React Native', () => {
       expect(posthog.getPersistedProperty(PostHogPersistedProperty.SessionStartTimestamp)).toEqual(nowMinus23Hour)
     })
   })
+
+  describe('flags endpoint with config', () => {
+    it('should call /flags with config=true when remote config is disabled', async () => {
```

Comment:
Without context, the test title is a little confusing. But I think I understand:if remote config is disabled, then `/flags` has to provide all the config values. If remote config is enabled, remote config provides those values so `/flags` doesn't have to, correct?

Reply:
yeah I'll clean up the name